src/docfx/config/ops/OpsConfigAdapter.cs

```diff
@@ -141,13 +143,31 @@ await _opsAccessor.GetDocumentUrls(), new[] { new { log_code = "", document_url
                     $"{PublicMetadataSchemaApi}{metadataServiceQueryParams}",
                 },
                 allowlists = AllowlistsApi,
+                allowedHTML,
                 trustedDomains,
                 sandboxEnabledModuleList = SandboxEnabledModuleListApi,
                 xref = xrefMaps,
                 isReferenceRepository = docsets.Any(d => d.use_template),
             });
         }
 
+        public static Dictionary<string, HashSet<string>?> ConvertAllowedHTML(string json)
+        {
+            var taxonomies = JsonConvert.DeserializeObject<Taxonomies>(json) ?? new();
+            if (taxonomies.TryGetValue(AllowedHtml, out var taxonomy))
```

Comment:
The trusted domain explicitly filters out `(empty)`, should this method do this too? or should be the taxonomy service be updated to not return `(empty)` at all? cc @OsmondJiang @hejingkan2005